source/types/retry.ts

```diff
@@ -33,4 +33,16 @@ export interface RetryOptions {
 	@default Infinity
 	*/
 	maxRetryAfter?: number;
+
+	/**
+	The upper limit of the `computedValue`.
```

Comment:
This should describe the option in simpler higher-level terms without mentioning `computedValue`.

Reply:
I copied from `got` a bit too literally as there is no `computedValue` field in the `RetryOptions` type, while in [got there is one](https://github.com/sindresorhus/got/blob/f6d1fb71b5e107ef4a6e3b5d5dfa2a80ec3f30a5/documentation/7-retry.md#calculatedelay).

I now rephrashed the comment.